conn.go

```diff
@@ -778,7 +836,7 @@ func (c *Conn) startHandshakeOutbound() {
 
 func (c *Conn) close() error {
 	if c.connectionClosed.Err() == nil && c.handshakeErr.load() == nil {
-		c.notify(alertLevelWarning, alertCloseNotify) //nolint
+		c.notify(alertLevelWarning, alertCloseNotify) // nolint
```

Comment:
Need to drop this change, causes a merge conflict b/c the code on master has been updated.